google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -192,7 +192,7 @@ clouddebug.warninglabel=Warning\:
 clouddebug.attachingtext=Attaching...
 clouddebug.stopandcontinue=Stop IDE Debugging.
 clouddebug.exitdebug=Exit debug mode and continue the debug session later.
-clouddebug.message.title=Cloud Debugger
+clouddebug.message.title=Stackdriver Debugger
```

Comment:
Yeah probably. I think the whole labelling strategy is a bit of a mess right now. I'm not going to fix this here though.